pkg/networkservice/common/heal/server.go

```diff
@@ -409,7 +227,7 @@ func (f *healServer) processHeal(ctx context.Context, request *networkservice.Ne
 		}
 	} else {
 		// Huge timeout is not required to close connection on a current path segment
-		closeCtx, closeCancel := clockTime.WithTimeout(ctx, time.Second)
+		closeCtx, closeCancel := context.WithTimeout(ctx, time.Second)
```

Comment:
Please don't use direct `context.WithTimeout`, `.WithDeadline` and `time.` methods. Use tools from `pkg/tools/clock`.